rotkehlchen/exchanges/bitfinex.py

```diff
@@ -654,23 +641,42 @@ def _query_currency_map(self) -> CurrencyMapResponse:
                     f'{self.name} currency map returned invalid JSON response. Check further logs',
                 )
             else:
-                currency_map = {
-                    bfx_symbol: symbol for bfx_symbol, symbol in response_list[0]
-                    if bfx_symbol not in set(BITFINEX_EXCHANGE_TEST_ASSETS)
-                }
-                with GlobalDBHandler().conn.read_ctx() as cursor:
-                    cursor.execute(
-                        'SELECT exchange_symbol, local_id FROM location_asset_mappings WHERE location IS ? OR location IS NULL;',  # noqa: E501
-                        (Location.BITFINEX.serialize_for_db(),),
+                # add the mappings in the globalDB
```

Comment:
so this is done to add possible assets that have no conflict on the symbol and aren't already in the db right? cAn you state that in the comment?